miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/errors/MiniAppError.kt

```diff
@@ -0,0 +1,16 @@
+package com.rakuten.tech.mobile.miniapp.errors
+
+import androidx.annotation.Keep
+
+/**
+ *  Contains the components to use custom errors from host app.
+ *  @property type The type of error.
+ *  @property message error message send to min app.
+ */
+@Keep
+data class MiniAppError(
```

Comment:
Maybe this type should be something like `MiniAppBridgeError` so it's more clear where it's used. What do you think?

Reply:
@corycaywood  yes, we can use `MiniAppBridgeError` instead of `MiniAppError`, May be JS SDK use `MiniAppError`, so i thought it may look consistent in both platform, i will update it no problem